sdk/storage/azure_storage_blob/src/lib.rs

```diff
@@ -8,7 +8,7 @@ mod generated;
 pub use generated::*;
 mod pipeline;
 
-// Moved from src/generated/mod.rs to surface these specific Options bags to the crate root
-// pub use crate::generated::clients::{
-//     BlobClientOptions, BlobContainerClientOptions, BlobServiceClientOptions,
-// };
+pub use crate::clients::{BlobClient, BlobContainerClient, BlobServiceClient};
+pub use crate::generated::clients::{
+    BlobClientOptions, BlobContainerClientOptions, BlobServiceClientOptions,
+};
```

Comment:
This brings both to the crate root, allowing for them to be directly imported as such:
`use azure_storage_blob::{BlobClient, BlobClientOptions, BlobContainerClient, BlobContainerClientOptions}`.

Reply:
Note that, currently the way this is, customers can import it either:
`use azure_storage_blob::clients::BlobClient;`
or
`use azure_storage_blob::BlobClient;`

Is that fine? If not, please suggest how to hide the other type of import. Thanks!